src/argilla/server/apis/v1/handlers/datasets/records.py

```diff
@@ -53,9 +47,17 @@
     TermsFilter,
     VectorSettings,
 )
-from argilla.server.schemas.v1.datasets import (
+from argilla.server.schemas.v1.records import (
     Record as RecordSchema,
```

Comment:
Why this `Record` class is the only one renamed to `RecordSchema`?

Reply:
It's because we have `Record` (as database model) used as type to be returned in some functions, at the same time we need to use `Record` (from schemas) as well.

Reply:
Okay. But we must unify this renaming convention. When a model-schema naming conflict occurs, the renamed class will be always the schema one